tests/phpunit/tests/REST_API/Stories_Autosaves_Controller.php

```diff
@@ -22,6 +22,15 @@
 use Spy_REST_Server;
 use WP_REST_Request;
 
+/**
+ * Class Stories_Autosaves_Controller
+ *
+ * @package Google\Web_Stories\Tests\REST_API
+ *          
+ * @covers Google\Web_Stories\REST_API\Autosaves_Controller
+ * @covers Google\Web_Stories\REST_API\Stories_Autosaves_Controller
+ * @covers Google\Web_Stories\REST_API\Template_Autosaves_Controller
```

Comment:
This should use `coversDefaultClass` I think? And also only cover `Stories_Autosaves_Controller`, not multiple classes.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/7225/commits/adcd74e078c2067aa8cea0aa95ca9acccef290b8